creedengojavascript.properties

```diff
@@ -0,0 +1,16 @@
+category=External Analysers
+description=Provide a list of static code analyzers (for Javascript language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
+homepageUrl=https://github.com/green-code-initiative/creedengo-javascript
+publicVersions=2.0.0
+archivedVersions=
+
+defaults.mavenGroupId=org.green-code-initiative
+defaults.mavenArtifactId=creedengo-javascript-plugin
+
+2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-javascript"
+2.0.0.sqs=[10.8,10.8.*]
```

Comment:
2025.1 was released yesterday. By not ending this range with `LATEST`, you restrict the plugin from being available there.